protein_ligand/stage1_submission_template.txt

```diff
@@ -0,0 +1,101 @@
+# Results for NanoLuc tranche1 virtual screening
+#
+# This file will be automatically parsed.  It must contain the following eight elements:
+# predictions, participant name, participant organization, name of method, software listing, method, method category, and ranked.
+# These elements must be provided in the order shown.
+# The file name must begin with the letters "NANOLUC" and then be followed by an underscore or dash.
+#
+# FILE FORMAT: All comment lines in this file (which begin with #) will be ignored.
+# Please use only UTF-8 characters in the non-comment fields. If your information (e.g. your name, etc.)
+# contains a non-UTF-8 character, you may note it in comments near that entry.
+
+
+# PREDICTIONS SECTION
+#
+# The data in each prediction line should be structured as follows:
+# Compound identifier, inhibition of luminescence (true/false)
+# Any compounds not listed are assumed to be predicted as nonbinders/non-inhibitors.
+# The list of predictions must begin with the "Predictions:" keyword, as illustrated here.
+Predictions:
+NCGC00019120-02, true
+NCGC00027933-02, false
+NCGC00034232-02, true
+NCGC00038620-02, false
+NCGC00040817-02, true
+NCGC00048081-02, false
+
+# PARTICIPANT INFORMATION SECTION
+#
+# Please list your name, using only UTF-8 characters as described above. The "Participant name:" entry is required.
+Participant name:
+David Mobley
+#
+# Please list your organization/affiliation, using only UTF-8 characters as described above.
+Participant organization:
+University of California, Irvine
+
+
+# METHOD NAME SECTION
+#
+# Please provide a brief (40 character limit) informal yet informative name of the method used.
+# Following is sample text; please edit to your taste.
+# The "Name:" keyword is required, as shown here.
+# 40 character limit.
+Name:
+OEDocking-template
+
+
+# SOFTWARE SECTION
+#
+# All major software packages used and their versions.
+# Please use a new line for each software.
+# Following is sample text; please edit to your taste.
+# The "Software:" keyword is required.
+Software:
+OpenEye docking toolkit v1.1
+
+
+# METHOD CATEGORY SECTION
+#
+# State which of the method category labels describe your prediction the best: `Docking`, `Ligand-based`, `MD`, `ML`, `Other`, `Null`.
+# If your method takes advantage of multiple approaches please report more than one category label, separated by comma.

```

Comment:
Does it matter if its a comma + single space or just a comma? (i.e. here it says comma separated, but example is comma + space)

This comment may be irrelevant. Not sure how well participants follow instructions/examples.

Reply:
Thanks. It should work either way; the parser splits on commas and then strips any spaces so it works whether they have them or not.